app/config/collections.php

```diff
@@ -2135,7 +2135,7 @@
                 '$id' => 'stderr',
                 'type' => Database::VAR_STRING,
                 'format' => '',
-                'size' => 16384,
+                'size' => 1000000,
```

Comment:
Im not sure of the size parameter. 

Is this the number of bytes or the number of characters ?

Reply:
Good question, I'll go into the Database library and see what it's setting

Reply:
it appears to be chars which can vary because of unicode

Reply:
There could be potentially 1 million characters but can also potentially only be 250k if they are using more unique characters that takes 4 bytes in UTF-8